lib/feed.js

```diff
@@ -263,6 +265,15 @@ Feed.prototype.query = function query_feed() {
       return self.retry();
     }
 
+    if(resp.statusCode == 404) {
+        destroy_response(resp)
+        self.log.warn('Database not found. Stopping changes feed after seq '+self.since)
```

Comment:
Should probably add spaces around the `+` to be consistent with the rest of the file (here and in the other changes).